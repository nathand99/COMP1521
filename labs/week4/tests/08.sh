echo 20 | /home/cs1521/bin/spim -file fac2.s | sed -e '1d;s/n  = //'
