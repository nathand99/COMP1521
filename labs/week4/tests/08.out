n! = -2102132736
